webapp/channels/src/packages/mattermost-redux/src/selectors/entities/threads.ts

```diff
@@ -94,7 +94,7 @@ export function makeGetThreadOrSynthetic(): (state: GlobalState, rootPost: Post)
                 reply_count: rootPost.reply_count,
                 participants: rootPost.participants,
                 last_reply_at: rootPost.last_reply_at ?? 0,
-                is_following: thread?.is_following ?? rootPost.is_following ?? null,
+                is_following: thread?.is_following ?? false,
```

Comment:
Does this mean that there may still be an issue anywhere else that `Post['is_following']` is used because it would be out of sync? I can't find any cases where it's used without checking if the post has replies in other ways, but perhaps that means we could remove its use on the frontend entirely and eventually remove it from the server type as well